salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountDetailsPage/LightningAccountDetailsPage.java

```diff
@@ -22,19 +22,18 @@
 public class LightningAccountDetailsPage extends BasePage implements IAccountDetailsPage {
 
     private static final int ACCOUNT_STRING_SIZE = 8;
-    private String accountInfoXpath = "//span[.='%1$s']/../../div[2]//%2$s";
-    private String opportunityToSearchXpath =
+    private static final String ACCOUNT_INFO = "//span[.='%1$s']/../../div[2]//%2$s";
+    private static final String OPPORTUNITY_TO_SEARCH =
             "//div[contains(@class,'normal')]//article[.//span[@title='Opportunities']]//a[contains(@href,'%s')]";
-
-    @FindBy(xpath = "//ul[@role='tablist']/li[@title='Details']/a")
-    private WebElement detailsTab;
+    private static final By DETAIL_TAB = By.xpath("//ul[@role='tablist']/li[@title='Details']/a");
+    private static final By SLA_EDITION_BUTTON = By.xpath("//button[@title='Edit SLA Expiration Date']");
 
     private void clickDetailsTab() {
-        GuiInteractioner.clickWebElement(detailsTab);
+        GuiInteractioner.clickWebElement(DETAIL_TAB);
     }
 
     private String getTextFromDetail(final String fieldName, final String tagType) {
-        return GuiInteractioner.getTextFromWebElement(By.xpath(String.format(accountInfoXpath, fieldName, tagType)));
+        return GuiInteractioner.getTextFromWebElement(By.xpath(String.format(ACCOUNT_INFO, fieldName, tagType)));
```

Comment:
Please, keep this By and Strings locators in lowerCase to be consistent to all the project, we use Upper case only for constants in primitive data type (int, Strings, char) but not applicable for locators.